app/page.tsx

```diff
@@ -9,12 +9,15 @@ import Tshirts from "./_assets/products/t-shirts.png";
 import Tumbler from "./_assets/products/tumbler.png";
 import UnderConstruction from "./_assets/under-construction.png";
 
-import Image from 'next/image'
+import Image from "next/image";
 
-import { Modal } from 'antd';
+import { Modal } from "antd";
 
 export default function Home() {
-  const [isModalOpen] = useState(typeof window !== 'undefined' && window.location.href !== "http://localhost:3000/");
+  const [isModalOpen] = useState(
+    typeof window !== "undefined" &&
+      window.location.href !== "http://localhost:3000/"
```

Comment:
can we 1 line this? should be okay as long as it's not 100 characters line in my opinion.